File: app/src/main/java/com/drewel/drewel/apicall/responsemodel/searchproductresponsemodel/Category.java

package com.drewel.drewel.apicall.responsemodel.searchproductresponsemodel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Category {

    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("category_name")
    @Expose
    private String categoryName;
    @SerializedName("img")
    @Expose
    private String img;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCategoryName() {
        return categoryName;
    }

    public void setCategoryName(String categoryName) {
        this.categoryName = categoryName;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

}

File: app/src/main/java/com/drewel/drewel/apicall/responsemodel/myorderdetailresponsemodel/Category.kt
package com.drewel.drewel.apicall.responsemodel.myorderdetailresponsemodel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Category {

    @SerializedName("id")
    @Expose
    var id: String? = null
    @SerializedName("category_name")
    @Expose
    var categoryName: String? = null

    @SerializedName("ar_category_name")
    @Expose
    var ar_category_name: String? = null
    @SerializedName("img")
    @Expose
    var img: String? = null

}
